api/MatterwikiAPI.js

```diff
@@ -0,0 +1,37 @@
+/* This is the wrapper for the Matterwiki API
+The main function in the wrapper is the .call(endpoint,type,body,token) which returns a
+promise with the response.
+*/
+
+var Promise = require("bluebird");
+var os = require("os");
+var hostname = os.hostname();
+import unirest from "unirest";
```

Comment:
How about using [`unfetch`](https://github.com/developit/unfetch)? I've generally seen `unirest` being used in server environments.